symengine/__init__.py

```diff
@@ -9,7 +9,8 @@
         UndefFunction, Function, FunctionSymbol as AppliedUndef,
         have_numpy, true, false, Equality, Unequality, GreaterThan,
         LessThan, StrictGreaterThan, StrictLessThan, Eq, Ne, Ge, Le,
-        Gt, Lt, GoldenRatio, Catalan, EulerGamma)
+        Gt, Lt, GoldenRatio, Catalan, EulerGamma, Dummy, perfect_power,
+        is_square, integer_nthroot, isprime, sqrt_mod, igcdex)
```

Comment:
Functions like `is_square` and `igcdex` are not pulled in when you do `from sympy import *`. I'd remove them from here.